.devcontainer/Dockerfile

```diff
@@ -0,0 +1,64 @@
+#-------------------------------------------------------------------------------------------------------------
```

Comment:
??

Reply:
This is from a merge. IIRC, @weshaggard checked it in. I didn't rebase because one or two of the earlier commits are a pain to resolve. Squashing this history will be faster/easier.